wordpress-mobile/WordPress-Android
# WP Stories: keep state

This PR provides support for Stories serialization / state keeping, particularly used when the StoryComposerActivity is sent to the background so we don't lose users work.

To test:
See steps in https://github.com/Automattic/portkey-android/pull/435

PR submission checklist:
